### DISCUSSION TITLE
How do we prevent Babel from compiling all language features?

### DISCUSSION THREAD

**Discussion by trusktr** [parcel-bundler/parcel#5103] [General] [answered]
Hello! I know it's a "feature" that Babel preset-env is used even if we don't specify it, but I really don't like it.

By default Parcel is compiling my code for some _really old browser_ like IE 11.

How do we tell Parcel to use only what's in Babelrc, and to not use preset-env?

**Comment by mischnic** [comment, marked as answer]
> not use preset-env?

That's unfortunately not possible with Parcel 1 (though Parcel 2 uses your babelrc without modification).

You should set a browserslist config: https://parceljs.org/javascript.html#default-babel-transforms

**Reply by trusktr** [reply]
Ah ok, thanks! That'll be very nice once Parcel 2 comes out.
